apps/api/src/variable/variable.e2e.spec.ts

```diff
@@ -204,6 +210,7 @@ describe('Variable Controller Tests', () => {
       expect(body.projectId).toBe(project1.id)
       expect(body.versions.length).toBe(1)
       expect(body.versions[0].value).toBe('Variable 3 value')
+      expect(body.versions[0].environment.slug).toBe(environment2.slug)
```

Comment:
This should point to `environment2.slug` since this version is being pointed to `environment2.id`